README.md

```diff
@@ -178,14 +178,6 @@ For details on enabling/configuring CORS, see
 To be able to meet the requirements of many organizations, Airflow supports many authentication methods,
 and it is even possible to add your own method.
 
-If you want to check which auth backend is currently set, you can use
-`airflow config get-value api auth_backends` command as in the example below.
-
-```bash
-$ airflow config get-value api auth_backends
```

Comment:
https://airflow.apache.org/docs/apache-airflow/2.11.0/configurations-ref.html#auth-backends

This isn't a thing in airflow 3 anymore